css/style.css

```diff
@@ -0,0 +1,184 @@
+body {
+  padding: 0 50px;
+  margin: 0;
+}
+
+.container {
+  cursor: pointer;
+}
```

Comment:
Prefer the use of link tags when needed instead of only mimicking a link.

Reply:
Noted